### PR TITLE
- add vaapi/system

### PR DESCRIPTION
**Pull request by SSE4**
Specify library name and version:  **add vaapi/system**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by ericLemanissier — APPROVED** [review]
all passed on https://github.com/bincrafters/system-packages-checks/actions/runs/533152448

**Comment by Croydon** [comment]
What is the reasoning behind this being a system package?

**Comment by SSE4** [comment]
this is a low-level API implemented by the video driver. basically, same case as mesa (GL API). this could be built from source, but implementation will be useless, as it won't be able to actually decode/encode anything.

**Comment by uilianries** [comment]
Not good, abandoned project: https://gitlab.freedesktop.org/freedesktop/freedesktop/-/issues/276

**Comment by ericLemanissier** [comment]
https://github.com/intel/libva does not seem abandonned

File: recipes/vaapi/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    license = "MIT"
+    settings = {"os": ["Linux", "FreeBSD"]}
+
+    def package_id(self):
+        self.info.header_only()
+
+    def _fill_cppinfo_from_pkgconfig(self, name):
+        pkg_config = tools.PkgConfig(name)
+        if not pkg_config.provides:
+            raise ConanException("VDPAU development files aren't available, give up")
```
Comment (uilianries) [inline recipes/vaapi/all/conanfile.py:21]:
```suggestion
            raise ConanException("VAAPI development files aren't available, give up")
```

Ctrl-C Ctrl-V

File: recipes/vaapi/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from conans.errors import ConanException
+
+
+class SysConfigVAAPIConan(ConanFile):
+    name = "vaapi"
+    version = "system"
+    description = "VA-API is an open-source library and API specification, which provides access to graphics hardware acceleration capabilities for video processing."
+    topics = ("conan", "vaapi", "hwaccel", "video")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/vaapi/"
```
Comment (ericLemanissier) [inline recipes/vaapi/all/conanfile.py:11]:
```suggestion
    homepage = "https://01.org/linuxmedia/vaapi"
```

File: recipes/vaapi/all/conanfile.py (hunk 1 above)
Comment (SSE4) [reply recipes/vaapi/all/conanfile.py]: I'll use self.name next time :)

**Review by ericLemanissier — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by intelligide — APPROVED** [review]

**Pull request merged**